docs/sources/write/links/_index.md

```diff
@@ -35,7 +35,27 @@ Although these other types of links still function, replace them with one of the
 
 ## Link from source content that's reused as multiple pages
 
-Use `ref` URIs.
+Use a full URL and ensure the mount target has `mount_links: true` in its front matter.
+The `mount_links` front matter tells Hugo to link to mounted content in the current project if it exists.
```

Comment:
I didn't realize this was a necessary step. Is this something that needs to be set for every mounted page? Also, I think an example of what this looks like in the front matter would be really helpful.

Reply:
It's something we need for every mount target, that is, a place that has content mounted into it. At the moment, that includes Grafana Cloud, GEM, GEL, and GET.

I can provide an example of using cascading front matter to set this as a default for all pages in a project.